packages/blip/app/components/chart/settingsDialog.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
```

Comment:
Can you confirm that this is the right copyright for newly added file in blip? Even the date?